arch/arc/Kconfig

```diff
@@ -104,6 +104,18 @@ config FP_FPU_DA
 
 menu "ARC CPU Options"
 
+config ARC_HAS_ZOL
+	bool "Use Zero Delay hardware loops"
```

Comment:
Should this really be user configurable.
Sounds a bit like something that should be promptless and selected based on soc/arc.

Reply:
Sounds reasonable, fixed for now. Thanks!